bench-tps/src/bench.rs

```diff
@@ -544,12 +545,23 @@ fn get_nonce_blockhash<T: 'static + BenchTpsClient + Send + Sync + ?Sized>(
     client: Arc<T>,
     nonce_account_pubkey: Pubkey,
 ) -> Hash {
-    let nonce_account = client
-        .get_account(&nonce_account_pubkey)
-        .unwrap_or_else(|error| panic!("{:?}", error));
-    let nonce_data = nonce_utils::data_from_account(&nonce_account)
-        .unwrap_or_else(|error| panic!("{:?}", error));
-    nonce_data.blockhash()
+    loop {
```

Comment:
I've been trying to figure this out, but best to hear it from you -- why is this loop needed? Is it just in case the funding of the nonces hasn't landed by the time you get here?

Reply:
Yes, I've found while testing this code on private cluster without this loop I often gen "Account Not Found" error. One way could be sleep for a while after funding these nonces accounts, but it is unclear for how long to sleep. So I decided to replicate the approach used for getting normal blockhash